shared/constants/chat2/message.js

```diff
@@ -166,10 +180,20 @@ const uiMessageToSystemMessage = (minimum, body): ?Types.Message => {
         team,
       })
     }
-    case RPCChatTypes.localMessageSystemType.complexteam:
-      return null
-    case RPCChatTypes.localMessageSystemType.createteam:
-      return null
+    case RPCChatTypes.localMessageSystemType.complexteam: {
+      const {team = ''} = body.complexteam || {}
+      return makeMessageSystemSimpleToComplex({
+        ...minimum,
+        team,
+      })
+    }
+    case RPCChatTypes.localMessageSystemType.createteam: {
+      const {team = '???', creator = '????'} = body.createteam || {}
+      return makeMessageSystemText({
+        text: new HiddenString(`${creator} created a new team ${team}.`),
```

Comment:
@mmaxim note it seems master just doesn't render these. on purpose?

Reply:
Nope this is a bug when @buoyad introduced the formatted system messages. This one didn't come along for the ride. Check out system/container.js

Reply:
k, they show now. doesn't match the designs but it shows now. leaving it for now